integration/testnet/network.go

```diff
@@ -468,10 +472,38 @@ func PrepareFlowNetwork(t *testing.T, networkConf NetworkConfig) *FlowNetwork {
 		bootstrapDir:       bootstrapDir,
 	}
 
+	// at-least 2 full access nodes must be configure in your test suite
+	// in order to provide a secure GRPC connection for LN & SN nodes
+	var (
+		accessNodeIDS strings.Builder
+		anCount = 0
+	)
+	for _, n := range confs {
+		if n.Role == flow.RoleAccess && !n.Ghost {
+			accessNodeIDS.WriteString(fmt.Sprintf("%s,", n.NodeID.String()))
+			anCount++
+		}
+	}
+	require.True(t,  anCount > 1, "at-least 2 access node that is not a ghost must be configured for test suite")
+
 	// add each node to the network
 	for _, nodeConf := range confs {
 		err = flowNetwork.AddNode(t, bootstrapDir, nodeConf)
 		require.NoError(t, err)
+
+		// string slice argument for LN/SN node --access-node-ids
+		// remove extra comma at the end of string
+		anIDS := accessNodeIDS.String()[:accessNodeIDS.Len()-1]
```

Comment:
Can use `strings.Join` to make this a bit cleaner

Reply:
https://github.com/onflow/flow-go/pull/1395/commits/63038258db5767cf410d7b0178d2c13bf2cf1e35